www/index.html

```diff
@@ -72,7 +72,10 @@
         </script>
 
         <script type="text/javascript" src="js/locate.js"></script>
-        <script type="text/javascript" src="js/strings.js"></script>
+        <script type="text/javascript">
+            add_script("js/strings."+CONFIG.LANGUAGE+".js");
```

Comment:
Presumably this does work okay, just wondered what would happen if it hadn't loaded before the time a string was needed, iyswim. Is there a nicer way this can be done, do you know?